radar-app-config/src/main/kotlin/org/radarbase/appconfig/persistence/HibernateVariableResolver.kt

```diff
@@ -163,7 +206,12 @@ class HibernateVariableResolver(
         scope: Scope,
         prefix: String,
     ): LongArray = createQuery(
-        "SELECT c.id FROM Config c WHERE c.scope = :scope AND c.clientId = :clientId AND c.name LIKE :prefix",
+        "SELECT c.id FROM Config c " +
```

Comment:
Would be nicer with Kotlin multiline string I think:
""""
SELECT...
WHERE...
AND...
"""